cbdr/cortex
# COR-683: Remember Me at Sign In

## Purpose:
Rebuilds the login form to not be two separate partials / two separate forms so that the "Remember Me" checkbox appears to be active when logging into both Legacy and Cortex. Basically condenses the two partials into one form and uses JQuery to toggle the necessary value for the hidden "Legacy" field.

Spoke with Justin, this matches the behavior he wants from this story.

> In the sign in screen, the Remember me toggle doesn't work on the Custom Content side. This leaves users confused - does it remember or doesn't it? Does it remember between Legacy and Custom Content?

## JIRA:
https://cb-content-enablement.atlassian.net/browse/COR-683

## Steps to Take On Prod
* Log out if currently logged in
* Choose either Legacy or Beta to log in to, then log in
* Confirm you logged into the correct Cortex, then log out
* Log into the opposite Cortex
* Confirm you logged into the correct Cortex

## Changes:
* Changes to setup
  * Condenses / Refactors the login form from two forms to one

* Architectural changes
  * N/A

* Migrations
  * N/A

* Library changes
  * N/A

* Side effects
  * Users without JS won't be able to log into Beta at this time; however they couldn't use it anyway...

## Screenshots
* Before
N/A

* After
N/A

## QA Links:
http://web.cortex-2.development.c66.me

## How to Verify These Changes
* Specific pages to visit
  * Login

* Steps to take
  * Log out if currently logged in
  * Choose either Legacy or Beta to log in to, then log in
  * Confirm you logged into the correct Cortex, then log out
  * Log into the opposite Cortex
  * Confirm you logged into the correct Cortex

* Responsive considerations
  * N/A

## Relevant PRs/Dependencies:
N/A

## Additional Information
This PR is basically tricking the User into thinking that a feature does something that it really doesn't, because that's how the feature is perceived. This was not possible with how the login form *was* set up, because we had two identical checkboxes both passing the same value, the only way to get this was to condense.
